src/hooks/useCurrentUser.ts

```diff
@@ -14,12 +13,18 @@ export interface CurrentUser {
 }
 
 export const useCurrentUser = (): CurrentUser => {
-  const { user, isAuthenticated } = useGoogleAuth();
+  const { accessToken } = useAuth();
   const { store } = useStore();
 
+  const user = accessToken.valid ? accessToken.user : null;
```

Comment:
I didn't touch this file much, but in the future it's worth figuring out if this should still live on the same way or be refactored further

Reply:
I ended up nuking the file :)